src/solver/simulation/ITimeSeriesNumbersWriter.h

```diff
@@ -3,13 +3,13 @@
 //
 
 #pragma once
-#include "antares/study/binding_constraint/BindingConstraintsRepository.h"
```

Comment:
This class is useless to me (see [this remark](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1219#discussion_r1229820466))